Assets/Scripts/Layers/LayerAsset.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using Netherlands3D.Twin.Layers.Properties;
+
+namespace Netherlands3D.Twin.Layers
+{
+    /// <summary>
+    /// A LayerAsset represents any asset, remote or embedded in the project, that is used in the
+    /// properties associated with a Layer. Each asset is unique for a whole layer, this naming collisions should
+    /// be taken into account.
+    ///
+    /// LayerAssets are used when saving the project to determine whether additional files should be embedded in the
+    /// project file, the URI scheme of these assets is 'project'. Remote assets are also supported.
+    /// </summary>
+    public class LayerAsset
+    {
+        /// <summary>
+        /// Remember the property data to which this asset belongs, so that we can include the identifier
+        /// when persisting the asset to disk.
+        /// </summary>
+        private readonly LayerPropertyData layerPropertyData;
```

Comment:
moet dit niet ILayerPropertyDataWithAssets zijn?

Reply:
Nee, want het belangrijke punt is de UUID die je eruit zou kunnen krijgen; en dat is een eigenschap van LayerDataProperty en niet ILayerPropertyDataWithAssets (en er is niks wat verplicht dat ILayerPropertyDataWithAssets  een instance is van LayerDataProperty)